feature/eks-cluster-creation/vars.tf

```diff
@@ -0,0 +1,64 @@
+// -- [ VARS ] ------------------------------------------------------
+// Cluster name
+variable "eks_cluster_name"     { default = "eks-terraform-cluster" }
+// Cluster version
+variable "eks_cluster_version"  { default = "1.17" }
+// AWS Region
+variable "region"               { default = "eu-central-1" }
+// Environment type (used to tag resources)
+variable "environment"          { default = "testing" }
+// AWS VPC name
+variable "vpc_name"             { default = "eks-terraform-vpc" }
+// AWS Resource group name
+variable "resource_group_name"  { default = "eks-terraform-rg" }
+// AWS Worker nodes name
+variable "worker_group1"        { default = "eks-worker-1" }
+variable "worker_group2"        { default = "eks-worker-2"}
+// VPC CIDR
+variable "cidr"                 { default = "10.0.0.0/16"}
```

Comment:
I think it is priettier to keep one style convention.
https://www.terraform.io/docs/configuration/style.html
https://www.terraform.io/docs/configuration-0-11/variables.html

Reply:
Sure! I will keep the old style for the future.